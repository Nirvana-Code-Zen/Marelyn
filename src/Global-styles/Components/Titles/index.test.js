import { H4 } from './index'
import renderer from 'react-test-renderer'

describe('<GlobalSytles/>', () => {
  test('Should be global styles ready', () => {
    const tree = renderer.create(<H4 />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
