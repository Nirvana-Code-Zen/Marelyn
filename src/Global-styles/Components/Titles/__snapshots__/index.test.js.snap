// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GlobalStytles /> Should be global styles ready  1`] = `
.c0 {
  font-size: 30px;
  text-align: center;
  text-align: center;
  font-family: "Didact Gothic",sans-serif;
  -webkit-letter-spacing: 4px;
  -moz-letter-spacing: 4px;
  -ms-letter-spacing: 4px;
  letter-spacing: 4px;
  color: #ffffff;
}

<h4
  className="c0"
/>
`;

exports[`<GlobalStytles/> Should be global styles ready  1`] = `
.c0 {
  font-size: 40px;
  font-family: "Didact Gothic",sans-serif;
  text-align: center;
  -webkit-letter-spacing: 4px;
  -moz-letter-spacing: 4px;
  -ms-letter-spacing: 4px;
  letter-spacing: 4px;
}

<h2
  className="c0"
/>
`;

exports[`<GlobalSytles/> Should be global styles ready  1`] = `
.c0 {
  font-size: 50px;
  font-family: "Didact Gothic",sans-serif;
}

<h1
  className="c0"
/>
`;
